### PR TITLE
Updating examples for BGP service communities

### PR DESCRIPTION
**Pull request by ganesr**
## Description
Updates example for Get-AzureRmBGPServiceCommunity.

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by ganesr** [comment]
@markjbrown Please let me know if this looks good. If it does, I will do the same for the others.

File: src/ResourceManager/Network/Commands.Network/help/Get-AzureRmBgpServiceCommunity.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿---
+
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/Get-AzureRmBgpServiceCommunity.md]: @ganesr please remove this space added at the top of the file

File: src/ResourceManager/Network/Commands.Network/help/Get-AzureRmBgpServiceCommunity.md [hunk 2]
```diff
@@ -16,15 +18,15 @@ Get-AzureRmBgpServiceCommunity [-DefaultProfile <IAzureContextContainer>] [<Comm
...
 
 ## DESCRIPTION
+This cmdlet provides a list of all services / regions, BGP communities, and associated prefixes.
 
 ## EXAMPLES
 
 ### Example 1
 ```
-PS C:\> {{ Add example code here }}
+Get-AzureRmBgpServiceCommunity
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/Get-AzureRmBgpServiceCommunity.md]: @ganesr would you be able to add the corresponding output (or a snippet of it if it's too big) from this cmdlet execution in this code block?

**Comment by ganesr** [comment]
@cormacpayne please review updates per your comments.

**Comment by cormacpayne** [comment]
@markjbrown these changes look good to me. I can merge the PR whenever you sign off as well.

**Pull request merged**